Gemfile

```diff
@@ -80,6 +80,7 @@ gem "snmp",                           "~>1.2.0",      :require => false
 gem "uglifier",                       "~>2.7.1",      :require => false
 gem "novnc-rails",                    "~>0.2"
 gem 'spice-html5-rails'
+gem "rugged",                         "~>0.23.0",     :require => false
```

Comment:
I don't know when people stopped going alphabetical in this list, but can it be moved up alphabetically?